nsqd/topic.go

```diff
@@ -275,6 +275,7 @@ func (t *Topic) messagePump() {
 				chanMsg.deferred = msg.deferred
 			}
 			if chanMsg.deferred != 0 {
+				atomic.AddUint64(&channel.messageCount, 1)
```

Comment:
But when the defered message finally published the messageCount do not increase.
```
    if chanMsg.deferred != 0 {
        atomic.AddUint64(&channel.messageCount, 1)
        channel.StartDeferredTimeout(chanMsg, chanMsg.deferred)
        continue
    }
```

**StartDeferredTimeout()** will  push the message into channel.deferredPQ and when the timeout message poped from channel.deferredPQ it will be published through **doRequeue()**. The messageCount hasn't changed in the process.
I reviewed the code so many times and verified it in practice